app/helpers/service_requests_helper.rb

```diff
@@ -57,7 +57,7 @@ def organization_description_display(organization)
 
   def ssr_name_display(sub_service_request)
     header  = content_tag(:span, sub_service_request.organization.name + (sub_service_request.ssr_id ? " (#{sub_service_request.ssr_id})" : ""))
-    header += content_tag(:span, '', class: 'glyphicon glyphicon-lock locked') if !sub_service_request.can_be_edited?
+    header += content_tag(:span, '', class: 'glyphicon glyphicon-lock locked') if !sub_service_request.can_be_edited? || sub_service_request.is_complete?
```

Comment:
@jayhardee9 
Can you take a look at the new changes? Because this logic is related to the editable statuses, I was able to move the logic into the can_be_edited? method.